### PR TITLE
SORQA-56

### PR DESCRIPTION
**Pull request by pk-sgent**
Fixes #

### CODE REVIEW

File: sormas-e2e-tests/src/main/java/org/sormas/e2etests/services/FollowUpVisitService.java [hunk 1]
```diff
@@ -110,4 +110,11 @@ public Visit buildSpecifiedFollowUpVisitForAvailableAndCooperative() {
         .sourceOfBodyTemperature(getRandomTemperature())
         .build();
   }
+
+  public Visit buildSpecifiedSymptoms() {
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/main/java/org/sormas/e2etests/services/FollowUpVisitService.java:114]: let's rename this to point only to temperature data, and pass the degrees as parameter so we can vary it in code however we need for further scenarios

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/FollowUpStep.java [hunk 2]
```diff
@@ -112,8 +112,17 @@ public FollowUpStep(
...
+          webDriverHelpers.clickOnWebElementBySelector(SAVE_BUTTON);
+          TimeUnit.SECONDS.sleep(2);
+        });
+
+    When(
+        "I save the Symptoms data",
+        () -> {
+          webDriverHelpers.scrollToElement(SAVE_BUTTON);
+          webDriverHelpers.clickOnWebElementBySelector(SAVE_BUTTON);
+          TimeUnit.SECONDS.sleep(2);
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/FollowUpStep.java:125]: same for this as above

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/FollowUpStep.java [hunk 3]
```diff
@@ -244,6 +295,12 @@ private void fillDateOfSymptoms(LocalDate dateOfSymptom) {
         DATE_OF_ONSET_INPUT, DATE_FORMATTER.format(dateOfSymptom));
   }
 
+  private void setSpecificSymptoms(String parameter) {
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/FollowUpStep.java:298]: the method sets a specific symptom, but inside covers 3, which are not all. Can we adjust it a bit? or maybe move out each field to make it reusable

**Review by rdutu-vg — CHANGES_REQUESTED** [review]

File: sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/FollowUpStep.java [hunk 4]
```diff
@@ -112,8 +112,17 @@ public FollowUpStep(
     When(
         "I save the Visit data",
         () -> {
-          webDriverHelpers.scrollToElement(SAVE_VISIT_BUTTON);
-          webDriverHelpers.clickOnWebElementBySelector(SAVE_VISIT_BUTTON);
+          webDriverHelpers.scrollToElement(SAVE_BUTTON);
+          webDriverHelpers.clickOnWebElementBySelector(SAVE_BUTTON);
+          TimeUnit.SECONDS.sleep(2);
```
[the remark this answers is not in the document]
Comment (pk-sgent) [reply sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/FollowUpStep.java:117]: That method is used by different test scenario, i am waiting there for spinner. It always need to wait 1-2 second to get it on the screen. It works in followup tests

File: sormas-e2e-tests/src/main/java/org/sormas/e2etests/services/FollowUpVisitService.java [hunk 5]
```diff
@@ -111,7 +111,7 @@ public Visit buildSpecifiedFollowUpVisitForAvailableAndCooperative() {
         .build();
   }
 
-  public Visit buildSpecifiedSymptoms() {
+  public Visit buildTemperatureOnlySymptoms() {
```
Comment (rdutu-vg) [inline sormas-e2e-tests/src/main/java/org/sormas/e2etests/services/FollowUpVisitService.java:114]: method parameter for temperature to reuse it

**Review by rdutu-vg — APPROVED** [review]

**Pull request merged**